lib/storage/data/LocationConstraintParser.js

```diff
@@ -1,6 +1,7 @@
 const { http, https } = require('httpagent');
 const url = require('url');
 const { S3Client } = require('@aws-sdk/client-s3');
+const { NodeHttpHandler } = require('@aws-sdk/node-http-handler');
```

Comment:
I think we want to use this one, that is the most up to date version (the aws sdk one is not maintained anymore it seems)

https://www.npmjs.com/package/@smithy/node-http-handler